### DISCUSSION TITLE
Importing rdkit on windows 10

### DISCUSSION THREAD

**Discussion by sajedehsaremi** [rdkit/rdkit#4416] [Q&A]
HI,
I am trying to get RDkit to work in order to use it for my QSAR models at work. I successfully installed RDkit and activate my environment on my conda through the following command on my Python 3 using windows 10:

conda install -c rdkit rdkit -y
conda activate PythonDataChem
now that I do have RDkit installed, I am trying to import my libraries as follow:
import rdkit
from rdkit import Chem
from rdkit.Chem import Descriptors, Lipinski

However, it keeps giving me the following error:

ModuleNotFoundError: No module named 'rdkit'

Can someone kindly advise how I can overcome this issue? FYI, I did update my python and it does support Python 3.6.

Many thanks in advance.

**Comment by greglandrum** [comment]
Hi @sajedehsaremi, this problem normally happens because you either aren't running python in the environment where you installed the RDKit.

How are you running Python? Are you using the command prompt or powershell or running it in some IDE?

An aside: we now recommend that you install the RDKit from conda-forge `conda install -c conda-forge rdkit`.

**Reply by sajedehsaremi** [reply]
Thank you @greglandrum  and @DavidACosgrove  for your response.

I am using IDE. I am going to activate my PythonDataChem environment, then use the conda-forge command, and hopefully, it will work this time.

**Comment by DavidACosgrove** [comment]
From the commands given, it doesn’t look like you’ve installed RDKit into
your PythonDataChem environment. Try activating the environment and then
installing RDKit.

On Thu, 12 Aug 2021 at 07:10, Greg Landrum ***@***.***> wrote:

> Hi @sajedehsaremi <https://github.com/sajedehsaremi>, this problem
> normally happens because you either aren't running python in the
> environment where you installed the RDKit.
>
> How are you running Python? Are you using the command prompt or powershell
> or running it in some IDE?
>
> An aside: we now recommend that you install the RDKit from conda-forge conda
> install -c conda-forge rdkit.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rdkit/rdkit/discussions/4416#discussioncomment-1161955>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACGF2FUCV6FP6AGQKZQZVW3T4NQVZANCNFSM5B762UTQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
-- 
David Cosgrove
Freelance computational chemistry and chemoinformatics developer
http://cozchemix.co.uk

**Reply by leoin841023** [reply]
Hi
I use conda to create a new environment for rdkit,
and use  conda install -c conda-forge rdkit to install 
but still have the above problem ---ModuleNotFoundError: No module named 'rdkit'


![1638249729842](https://user-images.githubusercontent.com/34364874/143990497-96abea83-bbc0-4199-a609-40d137fdc565.jpg)

**Reply by greglandrum** [reply]
This is almost always because the environment you are using is not the one the RDKit is in installed in.
Please be 100% sure that you are running the python (or ipython or jupyter or whatever) from the environment which has the RDKit installed. If you are using an IDE like pycharm or VS Code, be sure that it is using the correct conda environment.
